ckanext/s3filestore/uploader.py

```diff
@@ -132,7 +132,6 @@ def upload_to_key(self, filepath, upload_file, make_public=False):
         try:
             s3.Object(self.bucket_name, filepath).put(
                 Body=upload_file.read(),
-                ACL='public-read' if make_public else self.acl,
```

Comment:
Why is this being dropped?

Reply:
this can be reverted, not planned to be her